src/applications/disability-benefits/all-claims/content/bddRedirect.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import moment from 'moment';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import { EBEN_526_URL, BDD_INFO_URL } from '../../constants';
+import { activeServicePeriods } from '../utils';
+
+export default ({ formData }) => {
+  const endDates = activeServicePeriods(formData).map(
+    sp =>
+      sp.dateRange.to ? moment(sp.dateRange.to) : moment().add(200, 'days'),
+  );
+  const oneEighty = moment().add(180, 'days');
+  const ninety = moment().add(90, 'days');
+
+  let headline = 'You’ll need to file your claim on eBenefits';
+  let content;
+  const eBenLink = (
```

Comment:
probably overkill to create utility components for this and the learn more link unless we think we'll use them in letters more than the 2 instances in this PR

Reply:
:thinking: I'm not sure. I _think_ the most important part is making the URL itself shared. Once we have that, we can display it however we want—either as a normal link or a "button" link without having to worry about changing it anywhere else it's used if we update the style later.

:man_shrugging: